en/docs/guides/identity-verification/idvp-ui.md

```diff
@@ -0,0 +1,307 @@
+# Onboarding the UI of an IDV Provider to the Identity Server
+
+The UI of an IDV provider is a combination of static and dynamically generated sections. The static sections are already baked into the UI implementation while the metadata for the dynamic sections is pulled from the extensions implementation in the Identity Server [git issue].
+
+The following sections of the console and the management console are generated dynamically.
+
+**Console**
+- IDV Provider templates page.
+- Configuration section in the IDV Provider creation wizard.
+- Settings section in the IDV Provider edit page.
+
+**Management Console**
+- Configuration Properties section in the create page of the IDV Provider.
```

Comment:
In the management console UI, the create page is shown as "Add". Will this confuse the user?

Reply:
Resolved